templates/repo/actions/runs_list.tmpl

```diff
@@ -6,31 +6,35 @@
 	</div>
 	{{end}}
 	{{range .Runs}}
-		<li class="item gt-df gt-py-4">
+		<li class="item gt-df gt-py-3">
 			<div class="issue-item-left issue-item-icon gt-df gt-items-start">
 				{{template "repo/actions/status" (dict "status" .Status.String "locale" $.locale)}}
 			</div>
-			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3">
+			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3 gt-gap-2">
```

Comment:
5 helper classes now .... maybe it's also good to put the styles into the "action-item-main" class, or if they are general enough then make them for "issue-item-main"

Reply:
Managed to share a few styles in https://github.com/go-gitea/gitea/pull/25601/commits/cb5f08433b621589c57b2a3644450eae349f3d51.